src/components/room-invitation-creation-modal.js

```diff
@@ -1,15 +1,21 @@
 import PropTypes from 'prop-types';
-import { Form, Modal } from 'antd';
 import Logger from '../common/logger.js';
+import { Form, Input, Modal } from 'antd';
 import { useTranslation } from 'react-i18next';
 import errorHelper from '../ui/error-helper.js';
-import EmailFormItem from './email-form-item.js';
 import React, { useEffect, useState } from 'react';
+import inputValidators from '../utils/input-validators.js';
 import RoomApiClient from '../api-clients/room-api-client.js';
 import { useSessionAwareApiClient } from '../ui/api-helper.js';
 
+const FormItem = Form.Item;
+const { TextArea } = Input;
+
 const logger = new Logger(import.meta.url);
 
+const normalizeEmails = value => value.toLowerCase();
+const extractEmails = value => value.match(/[^\s;,]+/g) || [];
```

Comment:
How about `const extractEmails = value => (value.match(/[^\s;,]+/g) || []).filter(item => item);`
just to take out possible altogether missing emails (if ever that is the case)?
I'm thinking that those don't fall under the "invalid email" category 🤔

Reply:
I had something similar first, then changed the regex. This regex prevents any empty string matches, so `filter(item => item)` would never take out any item.

Reply:
Ah I see, smart regex, smart.